src/typings/azdataGraph.d.ts

```diff
@@ -0,0 +1,9 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+declare class azdataQueryPlan {
```

Comment:
You should ideally have the typings in the azdataGraph repo

Reply:
sounds like a reasonable improvement in a follow-up